src/custom-elements/src/components/progress/progress.tsx

```diff
@@ -0,0 +1,52 @@
+import { Component, Prop, Watch } from '@stencil/core';
+
+@Component({
+  tag: 'chi-progress',
+  styleUrl: 'progress.scss',
+  scoped: true
+})
+export class Progress {
+  /**
+   *  to set the value of a progress bar.
+   */
+  @Prop({ reflectToAttr: true }) value: string;
+
+  /**
+   *  to set the min value of a progress bar.
+   */
+  @Prop({ reflectToAttr: true }) min = '0';
```

Comment:
Since standard component does not have this attribute, do you think is necessary?

Reply:
@SergioQCL nope, I'll take it out.